src/pages/saksbehandling/steg/beregning/beregningUtils.ts

```diff
@@ -26,3 +27,11 @@ export const fradragstypeResourceId = (f: Fradragstype): string => {
             return 'fradrag.type.forventetinntekt';
     }
 };
+
+export const erGyldigBehandlingsstatus = (behandling: Behandling) =>
```

Comment:
Nitpick: jeg føler "gyldig" er et relativt begrep som her bare gir mening dersom man ser på hvor den brukes.
Kunne den ha hett noe ala
```suggestion
export const erIGyldigStatusForÅKunneBeregne = (behandling: Behandling) =>
```
?

Reply:
Yes, true!